tiles-request-api/src/main/java/org/apache/tiles/request/locale/URLApplicationResource.java

```diff
@@ -44,10 +42,45 @@
 public class URLApplicationResource extends PostfixedApplicationResource {
     private static final Logger LOG = LoggerFactory.getLogger(URLApplicationResource.class);
 
+    /**
+     * System parameter to specify additional remote protocols. If a url has a remote protocol, then any
+     * {@link IOException} will be thrown directly. If a url has a local protocol, then any {@link IOException}
+     * will be caught and transformed into a {@link FileNotFoundException}.
+     */
+    static final String REMOTE_PROTOCOLS_PROPERTY = "tiles.remoteProtocols";
+    private static final Set<String> REMOTE_PROTOCOLS;
+
+    static {
+        REMOTE_PROTOCOLS = initRemoteProtocols();
+    }
+
+    static Set<String> initRemoteProtocols() {
+        Set<String> remoteProtocols = new HashSet<String>();
+        remoteProtocols.add("ftp");
+        remoteProtocols.add("http");
+        remoteProtocols.add("https");
+        remoteProtocols.add("mailto");
+        remoteProtocols.add("netdoc");
+
+        String protocolsProp = getProperty(REMOTE_PROTOCOLS_PROPERTY);
+        if (protocolsProp != null) {
+            for (String protocol : protocolsProp.split(";")) {
+                remoteProtocols.add(protocol.trim());
```

Comment:
this is adding a feature to the code. it will need a bit more documentation, and be a bit clearer in the issue and commit message, as currently it's titled as a bug fix.